napari/plugins/_tests/test_utils.py

```diff
@@ -40,21 +40,13 @@ def test_get_preferred_reader_no_extension():
     assert reader is None
 
 
-def test_get_potential_readers_finds_npe1(mock_npe2_pm):
-    pth = 'my_file.tif'
-    readers = get_potential_readers(pth)
-    assert 'builtins' in readers
-
-
 def test_get_potential_readers_gives_napari(mock_npe2_pm, tmp_reader):
     pth = 'my_file.tif'
 
     tmp_reader(mock_npe2_pm, 'napari', ['*.tif'])
     readers = get_potential_readers(pth)
     assert 'napari' in readers
-    assert 'builtins' in readers
     assert readers['napari'] == 'napari (npe2)'
-    assert readers['builtins'] == 'builtins (npe1)'
```

Comment:
I think here we would be better off reverting #4575 (as we do in #4580). This PR supersedes #4580